<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WDLogin</class>
 <widget class="QMainWindow" name="WDLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>375</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>375</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>登陆</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLogin">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>登陆</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cAutoLogin">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>记住我</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eUserName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>用户 ID</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ePassword">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="lTitle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>聊天程序</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRegister">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>121</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>新用户注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNetowkStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>当前暂时无法连接 Mailto Server Service，请稍后再试。</string>
    </property>
   </widget>
  </widget>
  <action name="actionss">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>版本 1.0 Universal</string>
   </property>
  </action>
  <action name="actionSwitch_Connection_Port">
   <property name="text">
    <string>切换 MailtoService 节点</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
